<?xml version="1.0"?>
<launch>
	<node name="reactive_control_node" type="reactive_control_triple_integrator" pkg="reactive_control" respawn="false" output="screen">
		<rosparam file="$(find reactive_control)/config/reactive_params.yaml"/>
	</node>
</launch>